tracing/otel/model/types.go

```diff
@@ -0,0 +1,46 @@
+package model
+
+import (
+	"time"
+
+	"github.com/kiali/kiali/tracing/otel/model/json"
+)
+
+// Trace is a list of spans
+type TraceMetadata struct {
+	TraceID           string        `json:"traceID"`
+	RootServiceName   string        `json:"RootServiceName"`
+	StartTimeUnixNano string        `json:"startTimeUnixNano"`
+	DurationMs        time.Duration `json:"durationMs"`
+}
+
+type TracingResponse struct {
+	Traces []TraceMetadata `json:"traces"`
+}
+
+type Span struct {
+	SpanID            string           `json:"spanID"`
+	StartTimeUnixNano string           `json:"startTimeUnixNano"`
+	DurationNanos     string           `json:"durationNanos"`
+	Attributes        []json.Attribute `json:"attributes"`
+	Status            json.Status      `json:"status"`
+}
+
+type SpanSet struct {
+	Spans   []Span `json:"spans"`
+	Matched int    `json:"matched"`
```

Comment:
Here's that "Matched" variable name again. Should this be "Matches" or "NumMatched" or something else?